libs/ledgerjs/packages/cryptoassets/src/crypto-assets-importer/importers/eip712/index.ts

```diff
@@ -36,7 +36,7 @@ export default EIP712;
 export const importEIP712v2 = async (outputDir: string) => {
   console.log("importing EIP712 V2....");
   try {
-    const [eip712, hash] = await fetchTokens<EIP712>("eip712_v2.json");
+    const [eip712, hash] = await fetchTokensFromCDN<EIP712>("eip712_v2.json");
```

Comment:
why don't we use CAL service to do that?

Reply:
The scope of the PR was to focus on tokens for now, no other reason 👍